core/src/main/java/tachyon/client/BlockOutStream.java

```diff
@@ -58,10 +60,12 @@
    * @param file the file the block belongs to
    * @param opType the OutStream's write type
    * @param blockIndex the index of the block in the file
+   * @param tachyonConf the TachyonConf instance for this file output stream.
    * @throws IOException
    */
-  BlockOutStream(TachyonFile file, WriteType opType, int blockIndex) throws IOException {
-    super(file, opType);
+  BlockOutStream(TachyonFile file, WriteType opType, int blockIndex, TachyonConf tachyonConf)
```

Comment:
In spark, client can `new SparkConf` and set it to `SparkContext`. 

In tachyon, we can `new TachyonConf` and set it to `TachyonFS`, so that `TachyonFS` serves as something like `TachyonContext`. 

What do you think about this?